VideoApp/Video-CommunityTests/Info.plist

```diff
@@ -17,6 +17,6 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>64</string>
+	<string>65</string>
```

Comment:
Probably not a problem but I'm curious why bundle version needed to be bumped?

Reply:
I'm not sure how this got bumped? If 64 is the proper version, we can reset it.

Reply:
I would say reset it to 64. When master is merged in it will automatically update to correct version.

Reply:
Master was merged into this branch yesterday?

Reply:
Master has 65. Hmmm that is strange. Is the `sdk-tests` base branch basically being used as a feature branch?

Reply:
That I'm not sure. I think we *should* create a `feature/4.0` feature branch and that is where we should be merging this work into.